src/test/java/teammates/test/cases/webapi/SendReminderEmailActionTest.java

```diff
@@ -70,25 +75,25 @@ protected void testExecute() throws Exception {
                 Const.ParamsNames.STUDENT_EMAIL, student1InCourse1.email,
         };
 
-        remindAction = getAction(submissionParams);
-        r = getJsonResult(remindAction);
+        sendReminderEmailAction = getAction(submissionParams);
+        result = getJsonResult(sendReminderEmailAction);
 
-        msg = (MessageOutput) r.getOutput();
+        msg = (MessageOutput) result.getOutput();
         assertEquals("An email has been sent to " + student1InCourse1.email, msg.getMessage());
 
         ______TS("Typical case: Send email to remind a student to register for the course from student list page");
```

Comment:
I don't think you mention where is the request come from here.

Check LINE70 also.